rules.routing.yml

```diff
@@ -49,6 +49,22 @@ entity.rules_reaction_rule.delete_form:
   requirements:
     _permission: 'administer rules+administer rules reactions'
 
+entity.rules_reaction_rule.enable:
+  path: '/admin/config/workflow/rules/reactions/enable/{rules_reaction_rule}'
+  defaults:
+    _controller: '\Drupal\rules\Controller\RulesReactionController::performReactionRuleOperation'
+    op: 'enable'
+  requirements:
+    _permission: 'administer rules+administer rules reactions'
```

Comment:
this is vulnerable to CSRF attacks. we should enable CSRF protection for these routes.